angular-workspace/projects/ni/nimble-angular/src/directives/table-column/text/tests/nimble-table-column-text.directive.spec.ts

```diff
@@ -0,0 +1,14 @@
+import { TestBed } from '@angular/core/testing';
+import { NimbleTableColumnModule } from '../../nimble-table-column.module';
+
+describe('Nimble table column text', () => {
```

Comment:
I think this is missing tests for each property in the directive.

Reply:
So, for better or worse, this was somewhat deliberate as its following a similar pattern to what is in place for other directives that are solely meant for being child objects of other components like `ListOption`. That said, I can attempt to change things such that we can test the `TableColumnText` by providing a fixture that hosts the elements within a 
`Table`.

Reply:
Added tests.